docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,61 @@
+# Kubebuilder v1 vs v2
+
+There are a number of changes in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder).
+
+## Common changes
```

Comment:
We can probably rename this to `Project Level Changes` and merge this with the `Kubebuilder` changes listed at the bottom ?

Reply:
The original intend is to say the common changes in CR, CT and KB.